### PR TITLE
[Task01] created cpp file and readme.md

### PR DESCRIPTION
**Pull request by Ann1ks**

### CODE REVIEW

**Comment by as005411** [comment]
Folder should be named as0053xx (xx is your number in group) not your username.

File: trunk/as005302/doc/readme.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        cout << i << " - y(t)=" << y << endl;
+    }
+}
+
+void nonLinearEquation(float a, float b, float c, float d, float u, float y, int n)
+{
+    cout << "Nonlinear equation:" << endl;
+    for (int i = 0; i <= n; i++)
+    {
+        y = a * y - b * y * y + c * u + d * sin(u);
```
Comment (as005411) [inline trunk/as005302/doc/readme.md:63]: Mistake in b*y*y. Should be y(t-1) instead of y(t).

**Review by as005411 — CHANGES_REQUESTED** [review]
Mistake in nonlinear equation.
Everything else is nice

**Review by as005411 — CHANGES_REQUESTED** [review]
.vs folder is redundant.
Delete this folder and you're good to go

**Review by as005411 — APPROVED** [review]
Good job. Well done

**Review by as005411 — APPROVED** [review]
This should be an example for all of your group. Nice work

**Comment by Ann1ks** [comment]
Thanks :)

**Review by idzm — APPROVED** [review]

**Pull request merged**